tests/testsuite/path.rs

```diff
@@ -868,14 +868,16 @@ fn override_and_depend() {
         .build();
     p.cargo("build")
         .cwd("b")
-        .with_stderr(
+        .with_stderr(format!(
             "\
+warning: skipping duplicate package `a2` found at `{}/b/../a/a2`
 [COMPILING] a2 v0.5.0 ([..])
 [COMPILING] a1 v0.5.0 ([..])
 [COMPILING] b v0.5.0 ([..])
 [FINISHED] dev [unoptimized + debuginfo] target(s) in [..]
 ",
-        )
+            p.root().to_string_lossy()
+        ))
         .run();
```

Comment:
What I put below should fix the current test failures:

```rust
    p.cargo("build")
        .cwd("b")
        .with_stderr(
            "\
warning: skipping duplicate package `a2` found at `[..]`
[COMPILING] a2 v0.5.0 ([..])
[COMPILING] a1 v0.5.0 ([..])
[COMPILING] b v0.5.0 ([..])
[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]
",
        )
        .run();
```

Reply:
Thank you!